### PR TITLE
Update the demo config file for include directive

### PR DESCRIPTION
**Pull request by mapengfei53**
## Summary

在full.conf 更新include指令的使用方式说明

## Details

更新include指令的使用方式说明，协助用户正确使用include加载配置。具体可参考[#1399](https://github.com/ossrs/srs/issues/1399)

### CODE REVIEW

**Comment by mapengfei53** [comment]
Resubmit the PR and close the previous submission. Make sure to maintain the markdown structure.

`TRANS_BY_GPT3`

File: trunk/src/app/srs_app_config.cpp [hunk 1]
```diff
@@ -2033,29 +2033,27 @@ srs_error_t SrsConfig::parse_options(int argc, char** argv)
...
     }
-    
+
     // transform config to compatible with previous style of config.
     if ((err = srs_config_transform_vhost(root)) != srs_success) {
         return srs_error_wrap(err, "transform");
     }
-    
+
+    if (test_conf) {
```
Comment (winlinvip) [inline trunk/src/app/srs_app_config.cpp]: This is not the content to be submitted for PR, it is a bugfix. Please do not submit content that is unrelated to the PR.

Please undo this part of the code and submit a new one. Each PR corresponds to a branch.

`TRANS_BY_GPT3`

File: trunk/conf/full.conf [hunk 2]
```diff
@@ -2113,6 +2113,48 @@ vhost stream.transcode.srs.com {
...
+#
+# If load a config file.
+#   include ./conf/vhosts/ossrs.com.conf;
+#
+# If multiple config files are loaded, multiple config files, separated by spaces.
+#   include ./conf/vhosts/ossrs.com.conf ./conf/vhosts/srs.com.conf;
+#
+# @see https://github.com/ossrs/srs/issues/1399
+# See include.conf ,include.vhost.conf, include.engine.ffhd.conf
+#############################################################################################
```
Comment (winlinvip) [inline trunk/conf/full.conf]: No need for this comment, just include it directly.

```
include include.vhost.conf;
```

These comments and the description in the issue are basically repetitive. Don't duplicate the documentation, as you won't remember to update it.

`TRANS_BY_GPT3`

File: trunk/conf/include.vhost.conf [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    http_remux {
+        enabled     on;
+        mount       [vhost]/[app]/[stream].flv;
+    }
+
+    transcode {
+        enabled     on;
+        ffmpeg      ./objs/ffmpeg/bin/ffmpeg;
+
+        include     ./conf/include.engine.ffhd.conf;
```
Comment (winlinvip) [inline trunk/conf/include.vhost.conf]: Just one level of include is enough in the example, there is no need to include further inside.

We only need to demonstrate very typical use cases in the demo, there is no need to attempt to showcase all capabilities.

Just remove the transcode configuration directly.

`TRANS_BY_GPT3`

Comment (mapengfei53) [reply trunk/conf/include.vhost.conf]: Okay

`TRANS_BY_GPT3`

**Review by winlinvip — APPROVED** [review]
